### PR TITLE
Rename Taobao.xml to Taobao.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Taobao.com-falsemixed.xml [hunk 1]
```diff
@@ -1,15 +1,9 @@
 <!--
-	For rules not causing false/broken or valid MCB, see Taobao.xml.
+	For rules not causing false/broken or valid MCB, see Taobao.com.xml.
 -->
-
 <ruleset name="Taobao.com (MCB)" platform="mixedcontent">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="support.taobao.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: Why did you removed this host?

Comment (ivysrono) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]:
```
	<rule from="^http://assets\.taobaocdn\.com/"
		to="https://s.tbcdn.cn/" />
		<test url="http://assets.taobaocdn.com/p/header/header-min.css" />
```
https://support.taobao.com use https://assets.taobaocdn.com , break pages.

Comment (ivysrono) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: If we redirect https?://assets.taobaocdn.com/ to https://s.tbcdn.cn/ , we can enable support.taobao.com

Comment (J0WI) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: Is this documented somewhere?

Comment (ivysrono) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: The rule is in Taobao_CDN.com.xml
What should I do? Add comments or enable both?

Comment (J0WI) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: `support.taobao.com` is currently commented out and documented with `Breaks menu styling`. Could you just clarify that?

Comment (ivysrono) [inline src/chrome/content/rules/Taobao.com-falsemixed.xml]: Done.

**Pull request merged**